internal/errors/resolver/pkg.go

```diff
@@ -34,8 +34,11 @@ Error: No Kptfile found at {{ printf "%q" .path }}.
 
 	//nolint:lll
 	deprecatedKptfileMsg = `
-Error: Kptfile at {{ printf "%q" .path }} has an old version ({{ printf "%q" .version }}) of the Kptfile schema. Please update the package to the latest format. See https://kpt.dev/installation/migration for more details.
+Error: Kptfile at {{ printf "%q" .path }} has an old version ({{ printf "%q" .version }}) of the Kptfile schema.
 `
+	instructionForV1alpha1ToV1 = `Please update the package to the latest format by following https://kpt.dev/installation/migration.`
```

Comment:
It seems a bit confusing to dynamically build the templates. I think it might be easier to use one template and rely on the templating functions to determine which message should be used.

Reply:
Also, please include the resulting output in PR description.

Reply:
Updated.